src/ProcessMaker/Nayra/Engine/EngineTrait.php

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
This change add support to load processes without executing them into the engine. The execution will be delayed and controlled by the start timer event.